migrations/cockroach/V1.55__auth_connector.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE adminapi.idp_configs ADD COLUMN auth_connector_base_url TEXT;
+ALTER TABLE adminapi.idp_configs ADD COLUMN auth_connector_provider_id TEXT;
+ALTER TABLE adminapi.idp_configs ADD COLUMN auth_connector_machine_id TEXT;
+ALTER TABLE adminapi.idp_configs ADD COLUMN auth_connector_machine_name TEXT;
+ALTER TABLE auth.idp_configs ADD COLUMN auth_connector_base_url TEXT;
+ALTER TABLE auth.idp_configs ADD COLUMN auth_connector_provider_id TEXT;
+ALTER TABLE auth.idp_configs ADD COLUMN auth_connector_machine_id TEXT;
+ALTER TABLE auth.idp_configs ADD COLUMN auth_connector_machine_name TEXT;
+ALTER TABLE management.idp_configs ADD COLUMN auth_connector_base_url TEXT;
+ALTER TABLE management.idp_configs ADD COLUMN auth_connector_provider_id TEXT;
+ALTER TABLE management.idp_configs ADD COLUMN auth_connector_machine_id TEXT;
+ALTER TABLE management.idp_configs ADD COLUMN auth_connector_machine_name TEXT;
```

Comment:
Maybe we should also add resourceowner for detail responses when we already change something on the query model